src/balances/balances.service.ts

```diff
@@ -88,4 +88,13 @@ export class BalancesService {
       fiatConversion: fiatConversion,
     };
   }
+
+  async getSupportedFiatCodes(): Promise<string[]> {
```

Comment:
I'm not sure of this business logic. For brevity I'm assuming here that both `USD` and `EUR` we'll be always available, instead of checking their presence in the array 🤔